doc/source/tutorial/getting_started.rst

```diff
@@ -33,86 +49,78 @@ You can create an array from scratch by supplying data, axes and optionally some
             [[1023, 1038, 1053],
              [756, 775, 793]]]
 
-    # define axes
-    age = Axis(["0-9", "10-17", "18-66", "67+"], "age")
-    sex = Axis(["F", "M"], "sex")
-    year = Axis([2015, 2016, 2017], "year")
-
-    # create LArray object
-    pop = LArray(data, [age, sex, year], meta=[("title", "population by age, sex and year")])
+    # create an LArray object
+    pop = LArray(data, axes=[age, sex, year])
     pop
 
-Here are the key properties for an array:
+You can optionally attach some metadata to an array:
 
-* Array summary : dimensions + description of axes
+.. ipython:: python
 
-    .. ipython:: python
+    # attach some metadata to the pop array
+    pop.meta.title = 'population by age, sex and year'
+    pop.meta.source = 'Eurostat'
 
-        pop.info
+    # display metadata
+    pop.meta
 
-# TODO: move this in a section of the tutorial (inspecting arrays)
+To get a short summary an array, type:
 
-* number of dimensions
```

Comment:
you remove this, from getting started but did not add it back in another section?